src/modules/Instabug.ts

```diff
@@ -381,7 +381,8 @@ export const setReproStepsConfig = (config: ReproConfig) => {
  */
 export const setUserAttribute = (key: string, value: string) => {
   if (!key || typeof key !== 'string' || typeof value !== 'string') {
-    throw new TypeError('Invalid param, Expected String');
+    console.error('Invalid param, Expected String');
+    return;
```

Comment:
Let's add more context here to help in debugging such issues.
```suggestion
    console.error('IBG-RN: Expected key and value passed to setUserAttribute to be of type string');
    return;
```